pkg/controllers/openshift-apiserver.go

```diff
@@ -45,10 +45,8 @@ func NewOpenShiftAPIServer(cfg *config.MicroshiftConfig) *OCPAPIServer {
 	return s
 }
 
-func (s *OCPAPIServer) Name() string { return "ocp-apiserver" }
-func (s *OCPAPIServer) Dependencies() []string {
-	return []string{"kube-apiserver", "openshift-crd-manager"}
-}
+func (s *OCPAPIServer) Name() string           { return "ocp-apiserver" }
```

Comment:
Nitpicking here, but shall we rename this `openshift-apiserver` to correspond with the component + repo name?

Reply:
+1 improving readability

Reply:
I'll address these changes in a following PR.